security/deployment.adoc

```diff
@@ -8,108 +8,122 @@
+deployment configuration. For example, to set an `ImageChangeTrigger` in a
+deployment configuration called `frontend`:
 
-image::secure_deployments.png["Secure Deployments", align="center"]
+----
+$ oc set triggers dc/frontend \
+    --from-image=myproject/origin-ruby-sample:latest \
+    -c helloworld
+----
 
 [discrete]
 [[security-deployment-further-reading-1]]
 ==== Further Reading
 
 - _{product-title} Developer Guide_
-** xref:{sourcedir}dev_guide/deployments/how_deployments_work.adoc#dev-guide-how-deployments-work[How Deployments Work]
-** xref:{sourcedir}dev_guide/deployments/basic_deployment_operations.adoc#triggers[Setting Deployment Triggers]
-** xref:{sourcedir}dev_guide/application_lifecycle/promoting_applications.adoc#dev-guide-promoting-applications[Application Life Cycle Management -> Promoting Applications Across Environments]
+** xref:../dev_guide/deployments/how_deployments_work.adoc#dev-guide-how-deployments-work[How Deployments Work]
+** xref:../dev_guide/deployments/basic_deployment_operations.adoc#triggers[Setting Deployment Triggers]
+** xref:../dev_guide/application_lifecycle/promoting_applications.adoc#dev-guide-promoting-applications[Application Life Cycle Management -> Promoting Applications Across Environments]
 
 [[security-deployment-secrets-configmaps]]
 == Secrets and ConfigMaps
 
 The `Secret` object type provides a mechanism to hold sensitive information such
-as passwords, {product-title} client configuration files, `dockercfg` files,
-private source repository credentials, and so on. Secrets decouple sensitive
-content from the pods. You can mount secrets into containers using a volume
-plug-in or the system can use secrets to perform actions on behalf of a pod.
+as passwords, {product-title} client configuration files, *_dockercfg_* files,
+and private source repository credentials. Secrets decouple sensitive content
+from pods. You can mount secrets into containers using a volume plug-in or the
+system can use secrets to perform actions on behalf of a pod.
+
+For example, to add a secret to your deployment configuration using the web
+console so that it can access a private repository:
+
+. Create a new {product-title} project.
+
+. xref:../../dev_guide/secrets.adoc#dev-guide-secrets[Create a secret] that
+contains credentials for accessing a private image repository.
 
-The `ConfigMap` object holds key-value pairs of configuration data that can be
-consumed in pods or used to store configuration data for system components such
-as controllers. `ConfigMaps` are similar to secrets, but designed to more
-conveniently support working with strings that do not contain sensitive
-information.
+. Create a deployment configuration.
+
+. On the deployment configuration editor page or in the *fromimage* page of the
+xref:../../architecture/infrastructure_components/web_console.adoc#architecture-infrastructure-components-web-console[web console], set the *Pull Secret*.
+
+. Click the *Save* button.
```

Comment:
I removed the xrefs and updated the steps to be more straight forward on how to use the web console to accomplish.